File: week_02/day_2/multiple_classes_lab/bus_lab_solution/bus_stop.rb
class BusStop

  attr_reader :name, :queue

  def initialize(name)
    @name = name
    @queue = []
  end 

  def add_to_queue(person)
    @queue.push(person)
  end

  def queue_length()
    return @queue.length()
  end

  def clear_queue()
    @queue.clear()
  end

end

File: week_11/day_2/multiple_classes_lab/lab_solution/src/main/java/BusStop.java
import java.util.ArrayList;

public class BusStop {

    private String name;
    private ArrayList<Person> queue;

    public BusStop(String name){
        this.name = name;
        this.queue = new ArrayList<>();
    }

    public int getQueueCount(){
        return this.queue.size();
    }

    public void addPerson(Person person){
        this.queue.add(person);
    }


    public Person removePerson(){
        return this.queue.remove(0);
    }
}
